File: coffee-shop-ui/src/app/models/product.ts
export class Product {
    title: string;
    description: string;
    pricePerKg: string;
    amount: number;
    totalPrice: string;
    imageUrl: string;

    constructor(title: string, description: string, pricePerKg: string, totalPrice: string, amount: number, imageUrl: string) {
        this.title = title;
        this.description = description;
        this.pricePerKg = pricePerKg;
        this.totalPrice = totalPrice;
        this.amount = amount;
        this.imageUrl = imageUrl;
    }
}

File: coffee-shop-backend/coffee-shop-backend/Entities/Product.cs
using System;
using System.Collections.Generic;

namespace coffee_shop_backend.Entities;

public partial class Product
{
    public int Id { get; set; }

    public string Title { get; set; } = null!;

    public string Description { get; set; } = null!;

    public string PricePerKg { get; set; } = null!;

    public string ImageUrl { get; set; } = null!;

    public int Amount { get; set; }

    public string TotalPrice { get; set; } = null!;
}
